packages/keymanager-server/package.json

```diff
@@ -23,8 +25,9 @@
     "build:lib:watch": "yarn run build:lib --watch",
     "build:typedocs": "typedoc --exclude src/index.ts --out typedocs src",
     "build:types:watch": "yarn run build:types --watch",
+    "check-build": "node -e \"(async function() { await import('./lib/index.js') })()\"",
     "check-types": "tsc",
-    "lint": "eslint --color --ext .ts src/ test/",
+    "lint": "eslint --color --ext .ts src/",
```

Comment:
Why not lint test anymore?

Reply:
There are no tests, it breaks when it tries to lint a non-existent or empty directory